SabreTools.Models/N3DS/CIA.cs

```diff
@@ -14,6 +14,11 @@ namespace SabreTools.Models.N3DS
     /// <see href="https://www.3dbrew.org/wiki/CIA"/>
     public class CIA
     {
+        /// <summary>
+        /// Offset records for each file in the CIA
+        /// </summary>
+        public List<CIAOffsets>? Offsets { get; set; }
```

Comment:
Please avoid using `List<T>` when possible in models. I prefer to use arrays in most cases. Here that would be `CIAOffsets[]?`.